### PR TITLE
feat(locale): add city names (de)

### PR DESCRIPTION
**Pull request by atheck**
Added city names for local `de`.

### CODE REVIEW

File: src/locales/de/address/city_name.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+  'Neuss',
+  'Neustadt an der Weinstraße',
+  'Neuwied',
+  'Norderstedt',
+  'Nordhorn',
+  'Nürnberg',
+  'Oberhausen',
+  'Offenbach am Main',
+  'Offenburg',
+  'Oldenburg (Oldenburg)|Oldenburg',
```
Comment (ST-DDT) [inline src/locales/de/address/city_name.ts:147]: This one looks odd to me.

**Comment by ST-DDT** [comment]
Do you have a source for these (in case we need to need to update it)?

**Comment by Shinigami92** [comment]
Please use the new PR title naming convention: https://github.com/faker-js/faker/blob/main/CONTRIBUTING.md#committing

File: src/locales/de/address/city_name.ts (hunk 1 above)
Comment (atheck) [reply src/locales/de/address/city_name.ts:147]: That's right. I'll fix that.

**Comment by atheck** [comment]
I took the first 200 cities from here: https://de.wikipedia.org/wiki/Liste_der_Gro%C3%9F-_und_Mittelst%C3%A4dte_in_Deutschland

**Review by Shinigami92 — APPROVED** [review]

**Review by pkuczynski — APPROVED** [review]

**Pull request merged**